codegen/src/main/java/software/amazon/awssdk/codegen/naming/DefaultNamingStrategy.java

```diff
@@ -292,6 +292,11 @@ public String getFluentEnumGetterMethodName(String memberName, Shape parentShape
         return getterMethodName;
     }
 
+    @Override
+    public String getExistenceCheckMethodName(String memberName) {
+        return String.format("has%s", Utils.capitalize(memberName));
```

Comment:
Can we write a test for a reserved member name? What I mean is a list and map named "build" or something else reserved. I think to be consistent with the setter/getter we'll need to use `rewriteInvalidMemberName`.

Reply:
Sure, can do.

Reply:
```
    "ExistenceCheckNamingResponse": {
      "type": "structure",
      "members": {
        "Build":{"shape": "ListOfStrings"},
```
yields:
```
    public interface Builder extends JsonProtocolTestsResponse.Builder, SdkPojo,
                                     CopyableBuilder<Builder, ExistenceCheckNamingResponse> {
        /**
         * Sets the value of the Build property for this object.
         *
         * @param build
         *        The new value for the Build property for this object.
         * @return Returns a reference to this object so that method calls can be chained together.
         */
        Builder build(Collection<String> build);

        /**
         * Sets the value of the Build property for this object.
         *
         * @param build
         *        The new value for the Build property for this object.
         * @return Returns a reference to this object so that method calls can be chained together.
         */
        Builder build(String... build);
```
😬 
Seems like those should render to `buildValue`. But I'm not going to fix this in this PR.

Reply:
Fixed in 50de534